src/index.ts

```diff
@@ -1,13 +1,44 @@
 import express from "express";
 import type { Express, Request, Response } from "express";
-
+// import type { Task } from "../DB/dbConfig";
+import prisma from "../DB/dbConfig";
 const app: Express = express();
-const port = parseInt(process.env.PORT ?? "3000");
+const port = parseInt(process.env.PORT ?? "7000");
 
-app.get("/", (req: Request, res: Response) => {
-  res.send("Hello World!");
+app.use(express.json());
+// test
+app.post("/tasks", async (req: Request, res: Response) => {
+  try {
+    const { title, description, completed } = req.body;
+    const isCompleted = completed === "true";
+    // Create a new task using Prisma Client
+    const newTask = await prisma.task.create({
+      data: {
+        title,
+        description,
+        completed: isCompleted,
+      },
+    });
+    res
+      .status(201)
+      .json({ message: "Task created successfully", task: newTask });
```

Comment:
Maintain response format. Consider using 'data' or 'result' instead of 'tasks' in the JSON response to enhance uniformity.